### PR TITLE
Various minor edits

### PR DESCRIPTION
**Pull request by franklywatson**
Noticed a few things that needed tidying up

### CODE REVIEW

File: docs/evm/guides/integrating-metamask.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ export const PREVIEWNET_PARAMS = {
         symbol: 'FLOW',
         decimals: 18,
     },
-    blockExplorerUrls: ['https://previewnet.flowdiver.io'],
+    blockExplorerUrls: ['https://evm.flowscan.io'],
```
Comment (franklywatson) [inline docs/evm/guides/integrating-metamask.md:32]:
```suggestion
    blockExplorerUrls: ['https://eth.flowscan.io'],
```

File: docs/evm/using.mdx [hunk 2]
```diff
@@ -30,13 +30,13 @@ Manual method: Add EVM on Flow as a custom network to MetaMask:
...
 | Name            | Value                                   |
-| --------------- | --------------------------------------- |
+|-----------------|-----------------------------------------|
 | Network Name    | Flow Previewnet                         |
 | Description     | The public RPC url for Flow Previewnet  |
 | RPC Endpoint    | https://previewnet.evm.nodes.onflow.org |
 | Chain ID        | 646                                     |
 | Currency Symbol | FLOW                                    |
-| Block Explorer  | https://previewnet.flowdiver.io         |
+| Block Explorer  | https://evm.flowscan.io                 |
```
Comment (franklywatson) [inline docs/evm/using.mdx:39]:
```suggestion
| Block Explorer  | https://eth.flowscan.io                 |
```

File: src/components/addNetworkButton.tsx [hunk 3]
```diff
@@ -47,7 +47,7 @@ export const AddNetworkButton = () => {
               symbol: 'FLOW',
               decimals: 18,
             },
-            blockExplorerUrls: ['https://previewnet.flowdiver.io/'],
+            blockExplorerUrls: ['https://evm.flowscan.io/'],
```
Comment (franklywatson) [inline src/components/addNetworkButton.tsx:50]:
```suggestion
            blockExplorerUrls: ['https://eth.flowscan.io/'],
```

File: docs/evm/networks.md [hunk 4]
```diff
@@ -39,4 +39,4 @@ EVM on Flow has the following public RPC nodes available:
 | RPC Endpoint    | https://mainnet.evm.nodes.onflow.org   |
 | Chain ID        | 747                                    |
 | Currency Symbol | FLOW                                   |
-| Block Explorer  | https://flowdiver.io                   |
+| Block Explorer  | https://flowdiver.io                   |
```
Comment (gregsantos) [inline docs/evm/networks.md:42]: Should this also be https://eth.flowscan.io

Comment (franklywatson) [reply docs/evm/networks.md:42]: Left this alone because @Aliserag made the edits for this in his PR

**Review by gregsantos — APPROVED** [review]

**Pull request merged**